osu.Game/OsuGame.cs

```diff
@@ -828,6 +828,16 @@ private void beatmapChanged(ValueChangedEvent<WorkingBeatmap> beatmap)
         {
             beatmap.OldValue?.CancelAsyncLoad();
             beatmap.NewValue?.BeginAsyncLoad();
+
+            if (Host.Window == null)
+                return;
+
+            // prevent weird window title saying please load a beatmap
```

Comment:
This comment can be removed I think, the `DummyWorkingBeatmap` check is self-explanatory / self-documenting and does not require elaboration IMO.